src/sagemaker/mxnet/estimator.py

```diff
@@ -99,8 +99,43 @@ def __init__(
                     * ``custom-image:latest``
 
             distributions (dict): A dictionary with information on how to run distributed
-                training (default: None). To have parameter servers launched for training,
-                set this value to be ``{'parameter_server': {'enabled': True}}``.
+                training (default: None). Currently we support distributed training with
+                parameter server and MPI [Horovod].
+                To enable parameter server use the following setup:
```

Comment:
```suggestion
                parameter servers and MPI [Horovod].
                To enable parameter servers, use the following setup:
```

Reply:
I don't think there is a plural form for the term "parameter-server". Just like "Horovod", It's a Proper Noun not a common noun.